data/mods/Aftershock/vehicles/vehicle_parts.json

```diff
@@ -289,6 +289,96 @@
     "type": "vehicle_part",
     "symbol": "H"
   },
+  {
+    "abstract": "afs_lightweight_foldable_frame",
+    "type": "vehicle_part",
+    "name": { "str": "ultralight foldable frame" },
+    "item": "afs_titanium_foldable_frame",
+    "location": "structure",
+    "color": "light_gray",
+    "durability": 100,
+    "symbol": "H",
+    "folded_volume": "2000 ml",
```

Comment:
Folding into two liters seems a bit far fetched. Make it around 10-15L

Reply:
vanilla folding frame is 2.5L, 10L would make it sorta useless in comparison, sure you want it that high?

Reply:
Yes. Most likely that the vanilla frame shouldnt be that small either.